style.css

```diff
@@ -709,24 +769,1231 @@ body {
     color: #7f8cff;
   }
 
-  .social {
-    margin-left: 20%;
-    margin-top: 10px;
+  .grid_container {
+    display: grid;
+    grid-template-columns: (4, 1fr);
+    grid-gap: 10px;
+    width: 90%;
+    height: auto;
+    margin-top: 80px;
+    background-color: #f7f7f9;
   }
 
-  .social img {
-    margin-right: 20px;
+  .grid_desktop {
+    display: grid;
+    grid-column: 1;
+    width: 95%;
+    margin-left: 5%;
+    border-radius: 20px;
+    height: 600px;
+    margin-bottom: 80px;
+    background-image: url(/Asset/arteto.svg);
+    background-size: 50%;
+    background-repeat: no-repeat;
+    background-color: #fff;
   }
```

Comment:
- [ ] Greate work implementing the design, looks really good :+1: However let's stick to Figma's design as much as possible as requested in the project requirement. In your "works" section on desktop view, the project images are a little bit different compared to Figma's design (see images), kindly check it.

### Your design
![Screenshot from 2023-03-30 10-44-18](https://user-images.githubusercontent.com/91142432/228824069-98faf8f5-e728-4102-9fbc-e100eecfd2f7.png)
### What it supposes to look like
![Screenshot from 2023-03-30 11-41-00](https://user-images.githubusercontent.com/91142432/228825269-615e3ec6-4da7-42e8-bddf-f6ec771a34e1.png)

This apply to other works project image as well.